e2e/testdata/attachments.dockerapp/docker-compose.yml

```diff
@@ -16,11 +16,11 @@ services:
     deploy:
       resources:
         limits:
-          memory: $memory
+          memory: ${memory}
```

Comment:
Any idea what changed/how this was passing before?

Reply:
I honestly don't know... It's totally weird...

Reply:
OK then ;-)